server/routes/products.js

```diff
@@ -8,26 +8,27 @@ const Product = require('../models/Product');
 
 const verifyToken = require('../middleware/verify');
 
-const scrapePriceTag = async () => { 
-	console.log("scraping every two minutes");
-	Product.find({}).exec(async function(err, allProducts){
-		if(err || !allProducts.length){
-			console.log("error: No products for cron");
+const scrapePriceTag = async (url) => {
```

Comment:
Just a question, is this function only used by the cron job to go through all the products? If so we probably don't need the url parameter

Reply:
I am not entirely sure, I didn't write this function but I was trying to remedy it erroring out with "url is undefined" etc

Reply:
Yes only used by cron job. I will update the url parameter not needed. Thanks!